consensus/src/types/primitives.rs

```diff
@@ -234,14 +237,30 @@ impl ssz_rs::Deserialize for U64 {
 
 impl ssz_rs::SimpleSerialize for U64 {}
 
+impl serde::Serialize for U64 {
```

Comment:
try to use derive, it should work, no need to implement.

Reply:
as we talked, this is not working with the default serde:Serialize derive